.github/workflows/Jirabot.yml

```diff
@@ -39,28 +39,20 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GHUB_JIRA_USER_MAP: ${{ vars.GHUB_JIRA_USER_MAP }}
           JIRA_ISSUE_PROPERTY_MAP: ${{ vars.JIRA_ISSUE_PROPERTY_MAP }}
+          JIRA_ISSUE_TRANSITION_MAP: ${{ vars.JIRA_ISSUE_TRANSITION_MAP }}
         run: |
             import os
             import re
             import json
             from jira.client import JIRA
 
-            def updateIssue(jira, issue, prAuthorEmail, propertyMap: dict, pull_url: str) -> str:
+            def updateIssue(jira, issue, prAuthorEmail : str, transitionMap: dict, propertyMap: dict, pull_url: str) -> str:
                 result = ''
 
                 statusName = str(issue.fields.status)
-                if statusName == 'Open':
-                    transition = 'Start Progress'
-                elif statusName == 'In Progress':
-                    transition = ''
-                elif statusName == 'Resolved':
-                    transition = 'Reopen Issue'
-                elif statusName == 'Closed':
-                    transition = 'Reopen Issue'
-                else:
-                    transition = ''
+                transition = transitionMap.get(statusName, None)
 
-                if transition != '':
+                if transition != None and transition != '':
```

Comment:
it's hard to tell, how's the transaction == None or '' case handled?

Reply:
If transition == None or '' that indicates we don't need to make any transitions.

Reply:
I understand what the conditions mean, I asked how that case is handled.
if the new table is empty, does that issue get logged anywhere?

Reply:
Ah sorry I didn't understand your question. No, that issue wouldn't get logged anywhere, but if the table were empty or didn't have the specified status we would default to None. To your point it is probably a good idea to add a log message if we hit None, an empty string though should still be treated as a valid no-op